judo-psm-generator-engine/src/test/java/hu/blackbelt/judo/psm/generator/engine/NorthwindTest.java

```diff
@@ -54,14 +54,15 @@ void testCreateApplication() throws Exception {
         psmModel = new Demo().fullDemo();
         File testOutput =  new File(TARGET_TEST_CLASSES, NORTHWIND_TEST);
         try (Log bufferedLog = new BufferedSlf4jLogger(log)) {
-            PsmGenerator.executePsmGenerationToDirectory(PsmGenerator.PsmGeneratorParameter.psmGeneratorParameter()
-                            .projectGenerator(PsmGenerator.createGeneratorContext(psmModel, "test-project",
+            PsmGenerator.generateToDirectory(PsmGeneratorParameter.psmGeneratorParameter()
+                            .generatorContext(PsmGenerator.createGeneratorContext(psmModel, "test-project",
                                     Arrays.asList(
                                             new File(TARGET_TEST_CLASSES, OVERRIDE_1).toURI(),
                                             new File(TARGET_TEST_CLASSES, OVERRIDE_2).toURI()),
                                     Arrays.asList(TestHelper.class),
                                     null,
                                     null,
+                                    null,
                                     null))
```

Comment:
Why not use a builder to prevent these arguments as well?

Reply:
Fixed